Common/GPU/thin3d.cpp

```diff
@@ -768,4 +768,15 @@ const char *Bugs::GetBugName(uint32_t bug) {
 	}
 }
 
+const char *PresentModeToString(PresentMode presentMode) {
+	switch (presentMode) {
+	case (PresentMode)0: return "NONE (bad)";
+	case PresentMode::FIFO: return "FIFO";
+	case PresentMode::IMMEDIATE: return "IMMEDIATE";
+	case PresentMode::MAILBOX: return "MAILBOX";
+	default:
+		return "COMBO";  // TODO: Hardcode all the combinations?
```

Comment:
If it's gonna stay 3 options, that's not that bad I suppose: ab, bc, abc (we already have a, b, c.)  If it'll have more, that'd start to get messy.

-[Unknown]